testinfra/requirements.txt

```diff
@@ -4,7 +4,7 @@
 #
 #    pip-compile --output-file requirements.txt requirements.in
 #
-ansible-lint==3.4.13      # via molecule
+ansible-lint==3.4.16      # via molecule
```

Comment:
In the spirit of keeping this PR to the minimum, would you mind stripping out the module upgrades that are unrelated to the `safety` module ?

Reply:
I could, but I was using `pip-compile` and I don't actually know if there's any weird breakages, so I'm letting that tool do it's magic. This is something one of the ops homies would need to comment on.

Reply:
FWIW when I did the same in the past I just manually removed all dependencies that were not strictly related to the module I was adding. I don't think there is a way to ask pip-compile to focus on a single module and leave the others alone.

Reply:
Yeah @dachary is correct here ;) Unfortunately the pip-compile story sucks for just updating one dependency. You could temporarily peg all the dependencies in the `requirements.in` file and then run it ... that's a hacky way around it. Or yeah, do what @dachary said and try to manually edit the commit changes using something like `git add -p`